doc/api/console.md

```diff
@@ -271,6 +289,8 @@ functionality was unintended and is no longer supported.*
 <!-- YAML
 added: v0.1.104
 -->
+* `message` {String}
```

Comment:
Actually… `message` is optional too, it would be good if you could fix that in the `### console.trace(…)` line